backend/mcp_backend/routers/mcp.py

```
@@ -0,0 +1,963 @@
+from mcp_backend.models.mcp_servers import MCPServers
+
+log = logging.getLogger(__name__)
+
+########################################
+#
+# Utility functions
+#
+##########################################
+
+
+async def create_mcp_client(url: str, headers: Optional[Dict] = None):
```

Ruff (0.12.2)

Unused function argument: `headers`

(ARG001)